low-level/message_handlers/service_msg_handler.py

```diff
@@ -40,6 +43,15 @@ class ServiceMsgHandler(ScheduledModuleThread, InternalMsgQ):
     MODULE_NAME = "ServiceMsgHandler"
     PRIORITY = 2
 
+    SITE_ID = "site_id"
+    CLUSTER_ID = "cluster_id"
+    NODE_ID = "node_id"
+    RACK_ID = "rack_id"
+    SYSTEMDWATCHDOG = "SYSTEMDWATCHDOG"
+    MONITORED_SERVICES = "monitored_services"
+
+    RESOURCE_TYPE = "node:sw:os:service"

```

Comment:
We can import these from framework.utils.conf_utils. Please check.

Reply:
Done.